app/konnect/configure/runtime-manager/runtime-groups/index.md

```diff
@@ -0,0 +1,134 @@
+must be of the same type. Currently, only Kong Gateway runtime types are supported.
+
+Every organization has one default runtime group. With an Enterprise subscription,
+you can configure two additional custom runtime groups in the same Konnect
+account to manage runtime instances and their configuration in any groupings
+you want.
+
+_[TBA: diagram]_
+
+## Default runtime group
+
+The default runtime group is the foundational group in Konnect. Every
+organization starts with one default group.
+
+This group can be renamed, but it can't be deleted, and its status as the default
+group can't be changed.
+
+### Application registration in the Dev Portal
+
+When publishing documentation from the Service Hub to the Dev Portal, you can
+publish API specs from any version -- no matter which runtime group the version
+is in. However, application registration is only supported for Service versions
+running in the default runtime group.
+
+So, if you want a Service version to be available for application registration,
+create the version in the *default* runtime group. This means that you can have a
+mix of Service versions running in different groups for different purposes.
+
+For example, you might have:
+
+* The latest version of a Service published to the Dev Portal with the latest
+spec, running in the *default* group with application registration enabled.
+
+* A previous version of the Service published to the Dev Portal,
+running in the *default* group, but with application registration disabled.
+
+* Another previous version of the Service, published to the Dev Portal, and
+running in a *custom* runtime group. This version would only have the documentation
+available, and developers would not have access to application registration for
+this version.
+
+Out of these three scenarios, only the Service version in the first scenario
+would be available for application registration.
+
+## Multiple runtime groups
+{:.badge .enterprise}
+
+In addition to the default runtime group, you can also configure two additional
+custom groups. Splitting runtime instances and configuration into multiple
+runtime groups provides multiple specialized environments in one account.
+
+Some common use cases for using multiple runtime groups include:
+
+* **Environment separation:** Split environments based on their purpose, such as
+development, staging, and production.
+* **Region separation:** Dedicate each runtime group to a region or group of
+regions. Spin up runtime instances in those regions for each runtime group.
+* **Team separation:** Dedicate each runtime group to a different team and share
+resources based on team purpose.
+
+[Set up and manage runtime groups &rarr;](/konnect/configure/runtime-manager/runtime-groups/manage)
+
+## Administration
+
+Access to each runtime group is configurable on a team-by-team basis using
+entity-specific permissions.
+
+[Administer teams &rarr;](/link)
+
+## Global configuration
```

Comment:
Update: If you login to the helium environment today, you'll also see Services and Routes.
This is near identical to the experience of Kong Manager.

The "Gateway Services" in Runtime Manager = Service Version Implementation in Service Hub

Reply:
I also think Global Config is an outdated term with the introduction of multiple RGs. 
```suggestion
## Runtime group configuration
```